koku/masu/database/report_manifest_db_accessor.py

```diff
@@ -113,6 +112,10 @@ def manifest_ready_for_summary(self, manifest_id):
         """Determine if the manifest is ready to summarize."""
         return not self.is_last_completed_datetime_null(manifest_id)
 
+    def number_of_files(self, manifest_id):
+        """Return the number of files processed in a manifest."""
+        return CostUsageReportStatus.objects.filter(manifest_id=manifest_id).count()
```

Comment:
nitpick:  This seems to be returning number of all files. we should probably update the docstring here not to match the one for `number_of_files_processed`

Reply:
Good catch! I'll update it.